packages/webapp/src/components/Navigation/Menus/styles.module.scss

```diff
@@ -0,0 +1,49 @@
+
+.menuRoot {
+  top: 8px;
+}
+
+.menuList {
+  padding: 0px;
+  width: 176px;
+  border-radius: var(--Spaces-1x, 4px) 0px var(--Spaces-1x, 4px) var(--Spaces-1x, 4px);
+  box-shadow: 0px 4px 4px 0px rgba(0, 0, 0, 0.25);
+}
+
+.drawerMenuPaper {
+  border-radius: var(--Spaces-4x, 16px) var(--Spaces-4x, 16px) 0px 0px ;
+  box-shadow: 0px -4px 4px 0px rgba(0, 0, 0, 0.10);
+}
+
+.drawerMenuList {
+  padding-top: 0px;
+  padding-bottom: 0px;
+}
+
+.drawerListSubheader {
+  background-color: var(--mainBackground);
+  text-align: right;
+}
+
+.drawerCloseX {
+  width: 24px;
+  height: 24px;
+  padding: 6px;
+}
+
+.menuItemRoot {
+  width: 100%;
+  padding: 16px 22px 16px 14px;
+  gap: 8px;
```

Comment:
NIT, but I think you'll have to override MUI's min-width: 36px on the icons to min-width: 0 so that the spacing between the icon and the label is actually 8px

Reply:
Cool will take a look!

Reply:
Thanks for this one -- I decided to keep the MUI min-width and remove gap. I am trusting that MUI has a reason for it and it works out to ~10px gap now. I got the feeling that Loic trusts us to make these minor judgements and that the designs aren't meant to be pixel perfect.Let me know if you think overriding is preferred this seemed like a way to have less lines of code haha.

Reply:
It shouldn't be too many lines of code I think, if you just add a class to `ListItemIcon` and set min-width: 0 to that class, does it work? That's how I've currently set it up in my branch in the PureSlideMenu component. I don't think it needs to be pixel perfect but I prefer to take control over the spacing rather than let MUI do it since they could change that styling in the future if that makes sense. Granted, they could change the entire styling and that would still break things, but if it's a small enough override I'd go for it